### PR TITLE
With beautiful hugo

### PR DESCRIPTION
**Pull request by VincentRouvreau**
@lesteve 
A first version with beautiful hugo theme.
I did a [preview](http://pages.saclay.inria.fr/vincent.rouvreau/paris-swc/).
I am not quite happy with the page instructor (mainly raw html). I did not find any satisfying theme with multi-authors page (with photos, github, email, link, ...).
I think this could be performed with authors.

On main page, we could also add a nice photo under the big title.

I will see after your comments to build it automatically.

### CODE REVIEW

File: content/page/instructors.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Our instructors"
+date: 2020-01-08T07:39:57+01:00
+draft: true
+---
+
+This page lists all the instructors who have consented to be listed publicly on
+our web site so far. If you are a badged Instructor whose details do not appear
+here, but you would like to be listed, please propose a
+[merge request](https://github.com/paris-swc/paris-swc.github.io).
```
Comment (lesteve) [inline content/page/instructors.md]: I guess this is a Pull Request in github jargon.

**Comment by lesteve** [comment]
Wow thanks a lot! I just had a look at the preview and this looks very good!

**Comment by lesteve** [comment]
> I am not quite happy with the page instructor (mainly raw html). I did not find any satisfying theme with multi-authors page (with photos, github, email, link, ...).

It could be better sure but this is already good enough I would say for the first version. Improvements can be done in a different PR.

> On main page, we could also add a nice photo under the big title.

There are some photos from some past workshops in Bartozs presentation IIRC (which is mentioned somewhere in the PR). We could also leave it for a different PR.

> I will see after your comments to build it automatically.

This would be nice indeed.

File: content/page/instructors.md (hunk 1 above)
Comment (VincentRouvreau) [reply content/page/instructors.md]: Good to me. I fixed it on a6e2ef7

**Comment by lesteve** [comment]
Closing in favour of #2.

**Comment by lesteve** [comment]
Before I forget, something that looks worth a look to automatically build our Hugo website with Github Actions: https://github.com/marketplace/actions/hugo-to-gh-pages

**Pull request closed without merging**